hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -143,32 +146,45 @@ public void setCandidateRoster(@NonNull final Roster roster, @NonNull final Hand
                 context.storeFactory().readableStore(ReadableRosterStore.class).getActiveRoster();
         final var activeRosterHash = RosterUtils.hash(sourceRoster).getBytes();
         final var candidateRosterHash = RosterUtils.hash(roster).getBytes();
+
+        final var aggregationCalculationStart = Instant.now();
         final var tssPrivateShares =
                 getTssPrivateShares(sourceRoster, maxSharesPerNode, tssStore, candidateRosterHash, context);
         final var candidateRosterParticipantDirectory = computeTssParticipantDirectory(roster, maxSharesPerNode, (int)
                 context.networkInfo().selfNodeInfo().nodeId());
+        final var aggregationCalculationEnd = Instant.now();
+        final var durationOfAggregation = Duration.between(aggregationCalculationStart, aggregationCalculationEnd)
+                .toSeconds();
+        tssMetrics.updateAggregationTime(durationOfAggregation);
+
+        // https://github.com/hashgraph/hedera-services/issues/16166
+        final boolean hasKeyMaterial = false;
+        if (!hasKeyMaterial) {
```

Comment:
This whole `if` block is a little outdated. Check `develop` for the most recent

Reply:
Oh, good catch. It's a leftover after fixing conflict, ty